src/clr-angular/utils/drag-and-drop/interfaces/client-position.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+export interface DragPosition {
+    x: number;
```

Comment:
Is this a coordinate or the distance dragged? A comment would be helpful here.

Reply:
Yes, it's coordinates on page. Will comment and make it clear.